lib/UnixConfigLoader.php

```diff
@@ -153,15 +157,15 @@ private function parseOption(string $option): array
         switch ($name) {
             case "timeout":
                 // The value for this option is silently capped to 5s
-                return ["timeout", (int) \min((int) $value * 1000, self::RES_TIMEOUT)];
+                return ["timeout", (int) \min((int) $value * 1000, self::DEFAULT_TIMEOUT)];
```

Comment:
Should probably also check for negative values?

Reply:
@kelunik changed, passing negative values don't overwrite current value (default or set through env)